server/planning/common.py

```diff
@@ -220,6 +220,17 @@ def get_coverage_type_name(qcode):
     return coverage_type.get('name', qcode)
 
 
+def remove_autosave_on_spike(item):
```

Comment:
We now have ignore/cancel/save and `openactionfromeditor` unlocks the item. Do we still need this method on the server?

Reply:
Only for those during bulk select and spike. Because as per monitoring behavior there are no prompts to 'ignore-cancel-save' in this scenario. And only in this bulk-spike mode 'original' will have the lock information as 'edit'. Hence that check in the function (remove_autosave_on_spike) to avoid redundant delete action. Otherwise, it will have to send unlock for all those bulk-selected items which are on work-queue from front end (thought that would be too many promises to be resolved when doing it in backend would be simple).